File: C_C++_Programs/Basic_Programs/Prime_Number.c
#include<stdio.h>

int main(){

    int user_num;

      printf("Enter Any Number To Check Number Is Print Or Not....");
      scanf("%d",&user_num);

      int isPrime=0;

      if(user_num == 1 || user_num == 0){
         isPrime=1;
      }else{
           for(int i=2; i<user_num; i++){
              if(user_num%i==0){
                  isPrime=1;
              }
           }
      }

      if(isPrime==0){
         printf("Number  Is Prime : %d",user_num);
        }else{
          printf("Number Is Not Prime : %d",user_num);
      }


}

File: Simple_Program/Day_2/Prime_Number.java
package Day_2;

import java.util.Scanner;

public class Prime_Number {
    public static void main(String[] args) {
        
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter Number To Check Number Is Prime Or Not : ");
        int n=sc.nextInt();

        int count=0;

        for(int i=2; i<=n/2; i++){
              if(n%i==0){
                count++;
                break;
              }
        }

        if(count==0){
            System.out.println("Number Is Prime : ");
        }else{
            System.out.println("Number Is Not Prime : ");
        }

    }
}
